packages/zk-certificates/contracts/VerificationSBT.sol

```diff
@@ -19,6 +19,9 @@ contract VerificationSBT is IVerificationSBT, IERC721 {
     // token id to dApp
     mapping(uint256 => address) public tokenIdToDApp;
 
+    string public name;
```

Comment:
I think it would be good to change the interface we implement to from `IERC721` to `IERC721Metadata`.

Reply:
https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#IERC721Metadata

Reply:
Changed it